api/src/routing.js

```diff
@@ -251,7 +235,7 @@ const ONLINE_ONLY_ENDPOINTS = [
 
 // block offline users from accessing some unaudited CouchDB endpoints
 ONLINE_ONLY_ENDPOINTS.forEach(url =>
-  app.all(routePrefix + url, authorization.offlineUserFirewall)
+  app.all(routePrefix + url, authorization.checkAuth, authorization.offlineUserFirewall)
```

Comment:
Do you think the `offlineUserFirewall` check should also check that the user is logged in? I can't think of when we'd want to allow online _or_ logged out users to access anything...

Reply:
Yea, I agree. 
I removed `authorization.checkAuth` altogether and put the logged in check in the other middlewares.